curiouslearning/FeedTheMonsterJS
# Feat: Refactor all the buttons on Level End screen from canvas to HTML  (FM-328)

# Changes
- Use HTML buttons into level end screen
- Move levelend-scene.ts to src/scenes/levelend-scene/levelend-scene.ts
- Created next button component

# How to test
- Open the start screen of FTM.
- Tap or click any part of the start screen to proceed to the next screen
- Select any level to play
- Finish the game
- Make sure the buttons are html and working properly

Ref: [FM-328](https://curiouslearning.atlassian.net/browse/FM-328)


[FM-328]: https://curiouslearning.atlassian.net/browse/FM-328?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
